basilisk/src/ast/interpreter/test21.c

```
@@ -0,0 +1,30 @@
+  display_value (1 << 2 + 4);
+  display_value (1*a + 4);
+  display_value ((1*2 + 4)/32.);
+  display_value (sqrt(2.));
+  const double b = 3;
+  display_value (2*b);
+  a = b;
+  display_value (a);
+
+  {
+    double a, b;
+    if (a)
```

Cppcheck (2.19.0)

[error] Uninitialized variable

(uninitvar)